src/types/chains.ts

```diff
@@ -78,6 +81,73 @@ export const getTokenIcon = (token: AcceptedToken) => {
   }
 }
 
+export const getTokenDecimals = (token: AcceptedToken): number => {
```

Comment:
Token decimals shouldn't be statically defined; they should be based on the token information pulled. I made mention of this in a comment above